src/managed/dir.proj

```diff
@@ -24,7 +24,7 @@
       <Output ItemName="buildCmdArgs" TaskParameter="BuildArgs" />
     </GetBuildArgsByFrameworks>
 
-    <Exec Command="$(DotnetToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
+    <Exec Command="$(DotnetSdkToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
```

Comment:
potential future clean-up we should just let msbuild handle the building of these projects instead of shelling out to dotnet build.